initfiles/bash/etc/init.d/setupPKI.in

```diff
@@ -73,7 +73,7 @@ fi
 for i in "" ${compArray[@]} ; do
     compName=""
     if [ "$i" != "" ]; then
-        compName=""${i#*_}
```

Comment:
I remember vaguely that you added "#*_" to deal with an issue in 160, would removing it break that fix?

Reply:
This I think was originally from your commit a149d5694c917a4cef114eb345919c15284d165c.  I think we talked about possibly changing it but I don't think we ever did.